Chapter09/recipe-05/cxx-example/account/CMakeLists.txt

```diff
@@ -15,16 +15,19 @@ add_library(
   account
   MODULE
   account.cpp)
+
 target_link_libraries(
   account
   PRIVATE
```

Comment:
It's probably a good idea to have it as `PUBLIC` in case somebody downstream wants to use the library in a C++ code rather than loading the python module.

Reply:
Thanks!